chapters/en/Unit 13 - Outlook/RMT.mdx

```diff
@@ -0,0 +1,41 @@
+## Overview
+
+### What is Retention Networks
+RetNet, or Retentive Network, is a foundational architecture proposed for large language models in the paper <paper: RetNet>. This architecture is designed to address key challenges in the realm of large-scale language modeling: Training Parallelism, Low-Cost Inference, and Good performance.
```

Comment:
I suggest either using footnotes or adding the link directly instead of "<paper: RetNet>"